iOSDemo/ViewControllers/DemoViewController.swift

```diff
@@ -3,10 +3,18 @@ import UIKit
 import SweetUIKit
 
 enum DemoItem: Int {
-    case edit, collection, searchableCollection
+    case
+        edit,
+        collection,
+        searchableCollection
```

Comment:
I don't remember seeing this way of formatting before. Might be good to keep it as one-line for consistency.

Reply:
ah looks like the indentation got fucked up but yeah, this is what I use all the time